src/main/java/time/TimeInCapitals/showtime/LocalTimeService.java

```diff
@@ -47,27 +55,40 @@ public String getTimeAndDate() {
 		return localTimeAndDate;
 	}
 
-	public String changeTheFirstLetterToUppercase(String word) {
-
-		String[] regexResults = word.split("_");
+	public void getDataFromFile() {
 
+		File file = new File("D:/java/europe_capitals.txt");
 		StringBuilder stringBuilder = new StringBuilder();
+		String dataFromFile = null;
 
-		if (regexResults.length == 1) {
+		try (BufferedReader buffer = new BufferedReader(new FileReader(file))) {
```

Comment:
Chciałbym, abyś spróbował przepisać logikę wczytywania danych na streamy.

Reply:
OK, a podpowiedziałbyś coś odnośnie tego, co tu jest źle? :)

Reply:
Nie ma nic źle sensu stricte, poza tym że w takiej formie pisało się 10 lat temu i proponuję, abyś zastosował nowszą składnię z Javy 8.

Reply:
Do zrobienia :)

Reply:
Z tego co widzę to chyba już to poprawiłem.

Reply:
Z tego co widzę to już nie mam tego w LocalTimeService.

> Chciałbym, abyś spróbował przepisać logikę wczytywania danych na streamy.
Czy w metodzie getDataFromFile która jest w TimeCapitalsConfig mam spróbować przepisać wczytywanie z pliku na stream? Czy chodzi o samo zaciąganie danych z pliku, czy jeszcze podział danych?
Dopytuję, ponieważ komentarz odnosi się do innej klasy i kod odczytu został wcielony do innej metody.